<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RulesWindow</class>
 <widget class="QWidget" name="RulesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rule</string>
  </property>
  <widget class="QComboBox" name="box_action">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="box_match">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_match">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_action">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_rst">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
